src/jobs/test-nut.yml

```diff
@@ -20,10 +20,21 @@ parameters:
     description: 'By default, the latest version of the standalone CLI will be installed. To install via npm, supply a version tag such as "latest" or "6".'
     default: ''
     type: string
+  sfdx_executable_path:
```

Comment:
two additional parameters exposed, executable path which will be used to set `TESTKIT_EXECUTABLE_PATH` below.
repo_tag, used to checkout a specific version of the plugin. Should ensure that plugin test code is a close to the version of the plugin used by sfdx_cli.